state/factory/workingset.go

```diff
@@ -382,6 +382,10 @@ func (ws *workingSet) pickAndRunActions(
 			if !ok {
 				break
 			}
+			if nextAction.GasLimit() > blkCtx.GasLimit {
```

Comment:
just wonder what if there are many actions in actpool belong to different account? it will still loop up one action per account, right?

Reply:
yes, that's exactly the benchmark is testing against. We should be able to tolerant at least 20,000 accounts at the same time.